usi/src/main/proto/usi.proto

```diff
@@ -9,10 +9,10 @@ option java_outer_classname = "USIProto";
 option java_package = "grpc";
 
 service USIService {
-  rpc GetPower(SwitchInfo) returns (Power) {}
-  rpc GetInterface(SwitchInfo) returns (Interface) {} 
-  rpc disconnect(SwitchInfo) returns (SwitchActionResponse) {}
-  rpc connect(SwitchInfo) returns (SwitchActionResponse) {}
+  rpc GetPower(SwitchInput) returns (Power) {}
```

Comment:
Looking at this more closely -- should the return be PowerResponse?  (Same with Interface in the next line). Those seem like they're a bit short and likely to cause confusion...